server/game/cards/07.1-DDeeds/DrBrianFoxworth.js

```diff
@@ -0,0 +1,42 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class DrBrianFoxworth extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.isInTownSquare() && !this.booted && !this.hasAttachment(),
+            match: this,
+            effect: ability.effects.cannotBeCalledOut()
+        });
+        this.reaction({
+            title: 'Dr. Brian Foxworth',
+            when: {
+                onShootoutCasualtiesStepStarted: () => this.location === 'play area'
+            },
+            repeatable: true,

```

Comment:
If that is the reasoning, I would not do it with the repeatable. That would let the ability to be repeated regardless if player decides to use the ability or not. And in case there is some other react that would prevent the discard, the ability would not be marked as used and player would be able to use it again.
Rather, I would reset the ability usage only in case player decided not to proceed with the ability.

```javascript
onNo: () => {
    this.resetAbilities();
}

Reply:
I forsee issues with granted abilities with this.
I've read about having the handler return false, would that work?
```js
onNo: () => return false
```

Reply:
No it would not. Granted abilities would be affected, so if you are concerned, it would require updating the `BaseCard.resetAbilities()` to take a predicate based on which the proper ability can be reset.

Reply:
Then how about limiting the potential side-effects by doing only a part of what resetAbilities() would do?
```js
onNo: () => this.abilities.reactions.forEach(reaction => reaction.resetAbilityUsage())
```

Reply:
What if there will be some React abilities granted by other cards? 😃 
But to be honest, I do not really care since this is Old Timer card.

Reply:
> What if there will be some React abilities granted by other cards? 😃

After consulting with Al Swearengen, we came up with the following:
```js
const DrReactTitle = 'Dr. Brian Foxworth';
...
            title: DrReactTitle,
...
                    onNo: () => this.abilities.reactions.filter(reaction => reaction.title === DrReactTitle).forEach(reaction => reaction.resetAbilityUsage()),
```